src/views/SalesInfo/index.tsx

```diff
@@ -1,53 +1,129 @@
 import React from 'react';
-import VisuallyHidden from '@reach/visually-hidden';
-import { Bar, BarChart, ErrorBar, Tooltip, XAxis, YAxis } from 'recharts';
+import 'styled-components/macro';
+import { CartesianGrid, Line, LineChart, Tooltip, XAxis, YAxis } from 'recharts';
 import { colors } from 'components/theme';
 
-import { ReactComponent as BuyWpoktSvg } from 'assets/icons/buy_wpokt_button.svg';
-
 import {
-  StyledButtonContainer,
+  PartnersContainer,
+  ResourcesLink,
+  SalesInfoStatCard,
   StyledChartContainer,
   StyledContentContainer,
   StyledContentTextContainer,
   StyledInfoCard,
+  StyledResourcesContainer,
 } from 'views/SalesInfo/components';
 import { Card, InnerCardContainer } from 'components/Cards';
+import { Flex } from 'components/Containers';
 import Spacer from 'components/Spacer';
 import { H1, H2, P1, P2 } from 'components/Typography';
 
-const data = [
-  { name: 'Day 1', max: 20000, min: 10000, error: [1000, 1000] },
-  { name: 'Day 2', max: 45000, min: 34000, errorNegative: [10000, 2000] },
-  { name: 'Day 3', max: 32000, min: 24000, errorNegative: [10000, 2000] },
-  { name: 'Day 4', max: 10000, min: 4000, error: [2000, 10000] },
-  { name: 'Day 5', max: 20000, min: 11000, error: [2000, 10000] },
-  { name: 'Day 6', max: 30000, min: 4000, error: [2000, 20000] },
-  { name: 'Day 7', max: 2000, min: 1000, error: [1200, 2000] },
-];
+interface IData {
+  price: string;
+  created_at: string;
+}
 
 const SalesInfo: React.FC = () => {
+  const [data, setData] = React.useState<IData[]>([]);
+
+  React.useEffect(() => {
+    fetchData();
+  }, []);
+
+  const fetchData = async () => {
+    const response = await fetch('http://localhost:3000/prices');
```

Comment:
Let's put this URL as an environment variable :).

Reply:
Fixed in f05d4e127a24f8cc544a2078d453e4d8df87bb6b